### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by fenryxo**
Ref #95

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+CHANGELOG
+=========
+
+0.7.0
```
Comment (tpazderka) [inline CHANGELOG.md:4]: I would prefer to keep this in a section `UNRELEASED` and change the number on release.

File: eidas_node/tests/constants.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 KEY_FILE = str(DATA_DIR / 'key.pem')
 CERT_FILE = str(DATA_DIR / 'cert.pem')
 NIA_CERT_FILE = str(DATA_DIR / 'nia-test-cert.pem')
+NIA_CERT_FILE_LATEST = str(DATA_DIR / 'nia-test-cert-latest.pem')
```
Comment (tpazderka) [inline eidas_node/tests/constants.py:9]: This doesn't seem to be used anywhere. Am I missing something?

Comment (fenryxo) [reply eidas_node/tests/constants.py:9]: It will be used in python-env.

Comment (tpazderka) [reply eidas_node/tests/constants.py:9]: Then it should be there and not here :/

Comment (fenryxo) [reply eidas_node/tests/constants.py:9]: Only the constant or also the file?

Comment (tpazderka) [reply eidas_node/tests/constants.py:9]: Both

File: CHANGELOG.md (hunk 1 above)
Comment (fenryxo) [reply CHANGELOG.md]: Fixed.

File: eidas_node/tests/constants.py (hunk 2 above)
Comment (fenryxo) [reply eidas_node/tests/constants.py]: Removed.

**Pull request merged**